src/components/Home/sideBar/SideBar.jsx

```diff
@@ -1,8 +1,50 @@
 import React from "react";
 import "./_sideBar.scss";
+import {
+  MdSubscriptions,
+  MdExitToApp,
+  MdThumbUp,
+  MdHistory,
+  MdLibraryBooks,
+  MdHome,
+  MdSentimentDissatisfied,
+} from "react-icons/md";
 
-const Sidebar = () => {
-  return <div className="border border-danger">sidebar</div>;
+const Sidebar = ({ isSidebarVisible, handleToggleSidebar }) => {
+  return (
+    <div className={!isSidebarVisible ? "sidebar" : "sidebar open"}>
+      <li onClick={handleToggleSidebar}>
```

Comment:
As its temporary, instead if adding it on every li, you add it to the div 😃

Reply:
changed prop name as per suggestion.